### PR TITLE
Post RM-7152: Add ChromeBinariesProvider test

### PR DESCRIPTION
**Pull request by iam-mholle**

### CODE REVIEW

File: Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
```
Comment (Stefan-Ilic) [inline Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs]: Missing license header

File: Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  public class ChromeBinariesProviderTest
+  {
+    [Test]
+    public void GetInstalledExecutable ()
+    {
+      var provider = new ChromeBinariesProvider();
+
+      var result = provider.GetInstalledExecutable();
+
+      Assert.That (result, Is.Not.Null);
```
Comment (Stefan-Ilic) [inline Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs]: I think this assert is unnecessary, as the test will fail regardless if `result` is null

File: Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      Assert.That (result.BrowserBinaryPath, Is.EqualTo (@"C:\Program Files (x86)\Google\Chrome\Application\chrome.exe"));
+      Assert.That (
+          result.DriverBinaryPath,
+          Is.StringMatching (
+              Regex.Escape (Path.GetTempPath()) + @"Remotion.Web.Development.WebTesting.WebDriver\\chromedriver\\chromedriver_v\d+\.\d+\.\d+\.\d+\\chromedriver.exe"));
+      Assert.That (result.UserDirectory, Is.Not.Null.Or.Empty);
```
Comment (Stefan-Ilic) [inline Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs]: We could assert the format of `UserDirectory` like in the assert above with `DriverBinaryPath`

File: Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs (hunk 1 above)
Comment (iam-mholle) [reply Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs]: Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/12/commits/cc21cb58b42c8d277dc5010ce41de8afefffb1e3.

File: Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs (hunk 2 above)
Comment (iam-mholle) [reply Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs]: Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/12/commits/cc21cb58b42c8d277dc5010ce41de8afefffb1e3.

File: Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs (hunk 3 above)
Comment (iam-mholle) [reply Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs]: Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/12/commits/cc21cb58b42c8d277dc5010ce41de8afefffb1e3.

**Comment by iam-mholle** [comment]
Added to master with https://github.com/re-motion/Trunk-SVN2Git/commit/3c1b946872e8e478e1bcf1cc9e912d7ea6ed3731.

**Pull request closed without merging**